### PR TITLE
Not all types of 'eval' function calls wrapped (close #260)

### LINKED ISSUE #260: Not all types of 'eval' function calls wrapped
We have to wrap:

```javascript
var temp = eval;

temp('some code');
```

to 

```javascript
var temp = eval === window.eval ? script => eval(__proc$Script(script)) : eval;

temp('some code');
```

### PR DESCRIPTION
**Pull request by churkin**
\cc @LavrovArtem @inikulin

### CODE REVIEW

**Comment by churkin** [comment]
\retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
Maybe append regression test ```foo.bar.eval = {}``` ?

**Comment by churkin** [comment]
@LavrovArtem  I think the unit test will be enough.

**Comment by LavrovArtem** [comment]
lgtm

**Comment by inikulin** [comment]
@churkin FPR?

**Comment by churkin** [comment]
FPR

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
FPR

**Comment by LavrovArtem** [comment]
lgtm

File: src/processing/script/transformers/eval-get.js [hunk 1]
```diff
@@ -16,32 +17,40 @@ export default {
 
     nodeTypes: [Syntax.Identifier, Syntax.MemberExpression],
 
-    condition: (node, parentNode) => {
+    condition: (node, parent) => {
```
Comment (inikulin) [inline src/processing/script/transformers/eval-get.js]: I think it's will be much cleaner if we'll have two separate transforms: one for the identifier and one for the member. You can call them `eval-get` and `window-eval-get` respectively. @churkin What do you think?

Comment (churkin) [inline src/processing/script/transformers/eval-get.js]: I like it

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
FPR

File: src/processing/script/transformers/index.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ import documentWriteTransformer from './document-write';
 import evalTransformer from './eval';
 import evalCallApplyTransformer from './eval-call-apply';
 import getEvalTransformer from './eval-get';
+import getWindowEvalTransformer from './window-eval-get';
```
Comment (inikulin) [inline src/processing/script/transformers/index.js]: It's `evalGetTransformer` and `windowEvalGetTransformer`. The word order is the same as in the filename

**Comment by inikulin** [comment]
\r-

**Comment by churkin** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**